smdebug/core/index_reader.py

```diff
@@ -351,11 +351,15 @@ def _is_event_file_present(self, file):
 
     def list_index_files(self):
         index_dirname = IndexFileLocationUtils.get_index_path(self.path)
-        index_files = list_files_in_directory(index_dirname)
+        index_files_regex = ".+\/\d+_(.+)\.(json|csv)$"
```

Comment:
Add a comment explaining the regex. I understand filtering so it only gets `.json` or `.csv`, but it's not as clear filtering capturing groups with an underscore in the middle. Just worried that someone will change the file naming format and we'll have a bunch of silently ignored files.

Reply:
Put comment. Made regex more relaxed so look for .json, .csv or .tfevents